src/test/java/org/opentripplanner/middleware/persistence/TripHistoryPersistenceTest.java

```diff
@@ -27,66 +29,74 @@
  * {@link org.opentripplanner.middleware.controllers.api.ApiController} are not themselves tested here.
  */
 public class TripHistoryPersistenceTest extends OtpMiddlewareTest {
+    private static final int limit = 3;
+    private static final String userId = "123456";
     private static final String TEST_EMAIL = "john.doe@example.com";
 
-    OtpUser user = null;
-    TripRequest tripRequest = null;
-    TripSummary tripSummary = null;
-    List<TripRequest> tripRequests = null;
+    private static OtpUser user = null;
+    private static TripRequest tripRequest = null;
+    private static TripSummary tripSummary = null;
+    private static TripSummary tripSummaryWithError = null;
+    private static List<TripRequest> tripRequests = null;
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        user = createUser(TEST_EMAIL);
+        tripRequest = createTripRequest(userId);
+        tripSummary = createTripSummary();
+        tripSummaryWithError = createTripSummaryWithError();
+        tripRequests = createTripRequests(limit, user.id);
+    }
+
+    @AfterAll
+    public static void tearDown() {
+        if (user != null) Persistence.otpUsers.removeById(user.id);
```

Comment:
Could we use `user.delete()` here? Or add a comment if we should not use `user.delete()`.

Reply:
I've added a comment, same issue as apiUser.delete(). Both fail if the Auth0 user is not deleted.

Reply:
Or, should there be an `apiUser.delete(boolean deleteAuth0User)` to deal with both the tests and the regular code?

Reply:
That would make sense, depends how much we want to update the code to accomodate tests, @landonreed thoughts?

Reply:
Oh, interesting, I didn't see this before my review, but came to the same conclusion. I think there are cases in the regular code where we may not want to delete Auth0User anyways, so the overloaded delete method is the best course of action in my view.

Reply:
Updated.